File: BACK_END_CAR/CAR_RENTAL_API/Entities/Car.cs
using AuthToken.Database;
using CAR_RENTAL_API.Enum;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace CAR_RENTAL_API.Entities
{
    public class Car
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public Brand Brand { get; set; }
        public Guid BrandId { get; set; }
        public Modeltb Model { get; set; }
        public Guid ModelId { get; set; }
        public string Description { get; set; } = string.Empty;
        public decimal PricePerDay { get; set; }
        public int ModalYear { get; set; }
        public int SeatingCapacity { get; set; }
        public string FuelType { get; set; }
        public List<string> CarImages { get; set; } = new List<string>();
        public int Bags { get; set; }
        public string Transmission { get; set; }
        public int Doors { get; set; }
        public int MileageLimit { get; set; } = 100;

        public bool IsDeleted { get; set; } = false;
    }
}

File: FRONT_END_CAR/src/app/Shared/models/car.ts
export interface Car {
  carId: string;
  modelName: string;
  description: string;
  pricePerDay: number;
  modalYear: number;
  seatingCapacity: number;
  fuelType: string;
  status: string;
  bags: number;
  transmission: string;
  doors: number;
  carImages: string[];
  brandId: string;
  modelId:string;
  brandName: string;
}

export class Car {

  carImages: string[];
  image: string;

  constructor() {
    this.carImages = [];
    this.image = '';
  }
}